tests/components/recorder/test_history_db_schema_32.py

```diff
@@ -29,58 +25,15 @@
     assert_multiple_states_equal_without_context,
     assert_multiple_states_equal_without_context_and_last_changed,
     assert_states_equal_without_context,
+    old_db_schema,
     wait_recording_done,
 )
 
-CREATE_ENGINE_TARGET = "homeassistant.components.recorder.core.create_engine"
```

Comment:
This logic ended up getting repeated more than I was expecting we were going to have to so I moved it into a common helper and make a wrapper to load any old schema